taichi/backends/metal/kernel_manager.cpp

```diff
@@ -510,7 +510,7 @@ class HostMetalCtxBlitter {
       char *device_ptr = base + ret.offset_in_mem;
       if (ret.is_array) {
         void *host_ptr = host_ctx_->get_arg<void *>(i);
```

Comment:
```suggestion
        void *host_ptr = host_ctx_->get_arg<void *>(i);
```
This should be redundant now.

Reply:
done